detectable/src/main/java/com/synopsys/integration/detectable/detectables/docker/DockerDetectableOptions.java

```diff
@@ -38,9 +39,10 @@
     private final String dockerPlatformTopLayerId;
 
     public DockerDetectableOptions(final boolean dockerPathRequired, final String suppliedDockerImage, final String suppliedDockerTar, final String dockerInspectorLoggingLevel, final String dockerInspectorVersion,
-        final Map<String, String> additionalDockerProperties, final String dockerInspectorPath, final String dockerPlatformTopLayerId) {
+        final Map<String, String> additionalDockerProperties, final String dockerInspectorPath, final String dockerPlatformTopLayerId, final String suppliedDockerImageId) {
```

Comment:
A total nit: I'd suggest placing suppliedDockerImageId near suppliedDockerImage in the method signature.